main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapActivity.java

```diff
@@ -862,6 +864,11 @@ public void onTap(final int latitudeE6, final int longitudeE6, final int x, fina
                 }
             }
 
+            if (key.startsWith(IndividualRouteLayer.KEY_INDIVIDUAL_ROUTE) && !isLongTap) {
+                toggleElevationChart(this, viewModel.individualRoute.getValue(), nonClickableItemsLayer, getString(R.string.individual_route));
+                return;
+            }
```

Comment:
This will cause that caches aren't clickable anymore if the individual route is below them. 
- So either change the `LinkedList<RouteItem> result` to some super type which can hold more than only route items. This would have the advantage that tracks would appear in the "clicked items popup".
- Or explicitly search for key `IndividualRouteLayer.KEY_INDIVIDUAL_ROUTE` in case `result.size() == 0`

Reply:
done (by creating a holder class `RouteOrRouteItem`)